docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,295 @@
+
+## Configure Prometheus to write to Grafana Mimir
+
+Make a choice based on whether or not you already have a Prometheus server set up:
+
+- For an existing Prometheus server:
+
+  1. Add the following YAML snippet to your Prometheus configuration file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir, based on what is defined in the existing `scrape_configs` configuration.
+
+  1. Restart the Prometheus server.
+
+- For a Prometheus server that does not exist yet:
+
+  1. Write the following configuration to a `prometheus.yml` file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+
+     scrape_configs:
+       - job_name: prometheus
+         honor_labels: true
+         static_configs:
+           - targets: ["localhost:9090"]
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Start a Prometheus server by using Docker:
+
+     ```bash
+     docker run --network=host -p 9090:9090  -v <absolute-path-to>/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
+     ```
+
+## Configure Grafana Agent to write to Grafana Mimir
+
+Make a choice based on whether or not you already have a Grafana Agent set up:
+
+- For an existing Grafana Agent:
+
+  1. Add the following YAML snippet to your Grafana Agent metrics configurations (`metrics.configs`):
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Grafana Agent will write metrics to Grafana Mimir, based on what is defined in the existing `metrics.configs.scrape_configs` configuration.
+
+  1. Restart the Grafana Agent.
+
+- For a Grafana Agent that does not exist yet:
+
+  1. Write the following configuration to an `agent.yaml` file:
+
+     ```yaml
+     server:
+       http_listen_port: 12345
+       grpc_listen_port: 54321
+
+     metrics:
+       wal_directory: /tmp/grafana-agent/wal
+
+       configs:
+         - name: agent
+           scrape_configs:
+             - job_name: agent
+               static_configs:
+                 - targets: ["127.0.0.1:12345"]
+           remote_write:
+             - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Grafana Agent writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Create an empty directory for the write ahead log (WAL) of the Grafana Agent
+
+  1. Start a Grafana Agent by using Docker:
+
+     ```bash
+     docker run --network=host  -v <absolute-path-to-wal-directory>:/etc/agent/data -v <absolute-path-to>/agent.yaml:/etc/agent/agent.yaml grafana/agent
+     ```
+
+## Query metrics in Grafana
+
+First install Grafana, and then add Mimir as a Prometheus data source.
+
+1. Start Grafana by using Docker:
+
+   ```bash
+   docker run --rm --name=grafana --network=host grafana/grafana
```

Comment:
This only works for docker on linux. The below should work on all dockers

```suggestion
   docker run --rm --name=grafana -p 3000:3000 grafana/grafana
```

Reply:
That was required in my linux setup to get it working with having the ingress hostname in my `/etc/hosts` file.
The other option which might be more portable is using the `--add-host=<ingress-host>:<cluster-external-ip>` option.

Reply:
Actually add-host doesn't work on Windows either: https://github.com/docker/for-win/issues/1455

Reply:
The most portable solution is to require a DNS resolve-able `ingress-host` fqdn.  But on linux it's just so much easier to put it into /etc/hosts

Reply:
For now I propose to primarily document the DNS resolvable way but add notes about /etc/hosts and --network-host for linux

Reply:
If the ingress is in local for testing, we can document to use localdev.me which always resolves to 127.0.0.1.

Reply:
Just tested without /etc/hosts:
```
docker run -p 9090:9090 --add-host=mimir.krajo:10.64.0.1 -v `pwd`/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
docker run --rm --name=grafana -p 3000:3000 --add-host=mimir.krajo:10.64.0.1 grafana/grafana
```

So no need for /etc/hosts, `--add-host` works fine.

Reply:
updated without `--network=host` on the mainline